.env.example

```diff
@@ -5,6 +5,9 @@ MODELS_REPO=memory
 STORAGE_REPO=memory
 DEFAULT_TIMEOUT_MS=300000
 
+# Persona Identifier lock expiration time in ms (default 30000)
+IDENTIFIER_LOCK_EXPIRATION_MS=30000
```

Comment:
@ToranSharma , Nit: So if it has a default value then we can comment out this line? 👀

Reply:
👍 changed